### PR TITLE
Install testing framework

### PR DESCRIPTION
**Pull request by johanbounce**
# [PT Story](https://www.pivotaltracker.com/story/show/171817855)
```
As a development team
In order to be able to make sure our code is doing what it is supposed to do
We would like to be able to write and run automated tests
```
## Changes proposed in this pull request:
* Install and configured cypress. Test passing for h1 field render. 
* Install and configured jest and enzyme
* Changes with scripts in package.json

## Packages/Gems added
- Cypress
- Jest and enzyme

## Screenshot
Skärmavbild 2020-03-18 kl  18 11 47

### CODE REVIEW

File: cypress/integration/userCanSeeFirstHeader.feature.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+describe("user views menus", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3001");
+  });
+
+  it("displays first header header", () => {
+    cy.get("h1").should("contain", "VAMOS TEAM 2");
```
Comment (JaimeCrz) [inline cypress/integration/userCanSeeFirstHeader.feature.js:7]: Can you add picture of the test passing? on the PR

File: cypress/integration/userCanSeeFirstHeader.feature.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+describe("user views menus", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3001");
```
Comment (JaimeCrz) [inline cypress/integration/userCanSeeFirstHeader.feature.js]: If you are using base Url as (http://localhost:3001)

test should be: cy.visit("/")   ??

File: cypress/integration/userCanSeeFirstHeader.feature.js (hunk 1 above)
Comment (johanbounce) [reply cypress/integration/userCanSeeFirstHeader.feature.js:7]: Done. Check PR.

File: cypress/integration/userCanSeeFirstHeader.feature.js (hunk 2 above)
Comment (johanbounce) [reply cypress/integration/userCanSeeFirstHeader.feature.js]: Fixed.

**Pull request merged**